### ISSUE TITLE
"Version '' does not exist" error

### ISSUE THREAD

**Issue by edgararuiz** [mlverse/pysparklyr#45]
When you try to install Databricks using the cluster ID, and the Host ID is invalid, this error appears:

```r
> Sys.setenv("DATABRICKS_HOST" = "https://rstudio-partner-posit-default.cloud.databricks.com/o=notvalid")
> pysparklyr::install_databricks(cluster_id = "0914-200113-rc8v8keb")
! Retrieving version from cluster '0914-200113-rc8v8keb'
✔ Checking if provided version is valid against PyPi.org
Error in `install_environment()`:
! Version '' does not exist
Run `rlang::last_trace()` to see where the error occurred.
```

**Solution** - Remove any parameters, and leading forward slashes, from the URL. In the package, we'll check for that going forward and provide clearer warnings

**If you do not have an invalid host, or, if after correcting the host, please comment on this issue to let us know**

**Comment by edgararuiz** [comment]
Related to #44

**Comment by edgararuiz** [comment]
Addressed in #46

**Comment by edgararuiz** [comment]
New version now on CRAN: https://github.com/mlverse/pysparklyr/tree/v0.1.1
